README.md

```diff
@@ -13,34 +13,55 @@ This is the accompanying code to the paper "Towards Automated Circuit Discovery
 
 ## Installation:
 
+You need Python 3.8+ and Poetry (https://python-poetry.org/docs/) to install ACDC. You also need to install system dependencies.
+
+### System Dependencies
+
+#### Ubuntu linux
+
 ```bash
-pip install git+https://github.com/neelnanda-io/TransformerLens.git cmapy torchtyping
-git clone git+https://github.com/ArthurConmy/Automatic-Circuit-Discovery.git
-cd Automatic-Circuit-Discovery
-pip install -e .
+sudo apt-get update && sudo apt-get install libgl1-mesa-glx graphviz build-essential graphviz-dev
 ```
 
-### Details
+See the [Dockerfile](./Dockerfile) for details.
+
+#### Mac OS X
+On Mac, you need to let `pip` (inside `poetry`) know about the path to the Graphviz libraries.
 
-This codebase currently works on top of the TransformerLens main branch (as of 6th June 2023), so probably `transformer_lens==1.2.3` or later (for now, you need to install TransformerLens from source, such as by using the above fast installation command).
+``` bash
+brew install graphviz
+export CFLAGS="-I$(brew --prefix graphviz)/include
+export LDFLAGS="-L$(brew --prefix graphviz)/lib
+```
```

Comment:
Can we move this to the end of the README? This would put me off using something if I had to do this first. We can mention that you can run ACDC without graphics (e.g acdc/main.py from commandline)

Reply:
@rhaps0dy I couldn't do this in just one suggestion, so I made [a PR here](https://github.com/ArthurConmy/Automatic-Circuit-Discovery/pull/49) to merge into this (`adria/poetry-makefiles`) branch

Reply:
I appreciate that thanks!